helm_deploy/apply-for-legal-aid/values-uat.yaml

```diff
@@ -117,6 +117,10 @@ omniauth_entraid:
   mock_auth_enabled: true
   redirect_uri: https://main-applyforlegalaid-uat.cloud-platform.service.justice.gov.uk/auth/entra_id/callback # overridden for branches (see deploy script)
```

Comment:
Do we need to amend bin/deploy to allow for branch logins too or do we switch to mock on branches?
Maybe this comment is for @agoldstone93 's branch? 🤔

Reply:
@colinbruce I was thinking of just mocking on UAT branches for that reason but I'm open to suggestions